### DISCUSSION TITLE
Add MaxTraffic and ExpirationDate command for FileAccessManager

### DISCUSSION THREAD

**Discussion by asoMansoury** [vpnhood/VpnHood#373] [General]
Hi
I need to generate a token by command line in ubuntu
The user I want to define must have below features
Max Client = 1;
Max Traffic = 100GB
Expiration Date = 2023/21/08

Please guide me to generate this user in ubuntu's command line

**Comment by trudyhood** [comment]
For command line help:
```sh
/opt/VpnHoodServer/vhserver gen /?
```

For your need:
```
/opt/VpnHoodServer/vhserver gen -maxClient 1
```
Max Traffic and ExpirationDate have been implemented for FileAccessManager but have yet to be published. hover it is minimal; for example, the traffic will not be reset automatically. I recommend you to use VpnHood Access Manager.
